packages/extension-chakra-storefront/src/components/icons/index.test.js

```diff
@@ -43,16 +42,32 @@ test('uses intl from props when rendered outside provider tree', () => {
         formatMessage: jest.fn()
     }
 
-    // render without providers
-    render(<Icons.LockIcon intl={mockIntl} />)
+    const Icons = icon(
+        'lock',
+        {
+            'aria-hidden': false,
+            focusable: true
+        },
+        {
+            'aria-label': defineMessage({
+                id: 'icons.assistive_msg.lock',
+                defaultMessage: 'Secure'
+            })
+        }
+    )
 
+    // render without providers
+    renderWithChakraProvider(<Icons intl={mockIntl} viewBox="0 0 40 40" />)
+    const svg = document.querySelector('.chakra-icon')
+    // confirm that the other prop are being render properly
     expect(mockIntl.formatMessage).toHaveBeenCalled()
-    expect(useIntl).not.toHaveBeenCalled()
+    expect(svg).toBeInTheDocument()
+    expect(svg).toHaveAttribute('viewBox', '0 0 40 40')
```

Comment:
Same with previous comment, we do not test for implementation, but the outcome we want to see after rendering the component. Here after we inject intl object as prop, we should expect to see the rest of props are passed into the svg element when rendering.